src/hb_cache.erl

```diff
@@ -188,6 +188,7 @@ list(Path, Opts) when is_map(Opts) and not is_map_key(<<"store-module">>, Opts)
             list(Path, Store)
     end;
 list(Path, Store) ->
+    % TODO: Fix seperate store requests
```

Comment:
When providing multiple stores, the calls `resolve` and `list` should be tied to the same store.
Currently, if Store2 contains the key/value and Store1 doesn't, `resolve` will return the same path when calling `list` on Store1, but not call `resolve` on Store2.